### PR TITLE
fix: amilajack/eslint-plugin-compat#224

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "caniuse-db": "^1.0.30001017",
     "lodash.memoize": "4.1.2",
     "mdn-browser-compat-data": "^1.0.3",
-    "semver": "^7.1.1"
+    "semver": "6.3.0"
```
Comment (ljharb) [inline package.json]:
```suggestion
    "semver": "^6.3.0"
```

**Comment by zloirock** [comment]
It's the breaking change in the minor semver release ( 😂), you could use `7.0.0`, https://github.com/npm/node-semver/commit/d61f828e64260a0a097f26210f5500e91a621828.

**Comment by ljharb** [comment]
engines is purely advisory; yarn breaking on it isn’t a breaking change, it’s a bug in yarn.

**Comment by zloirock** [comment]
Since it's not fixed for a long time in Yarn, it's positioned as a feature, not as a bug, so it's a breaking change in `semver` since Yarn is an important component of the NPM ecosystem and adding this field is the break of compatibility with Yarn + Node version less than specified -> breaking change, even if it defers from the official NPM position about this field. However, I don't care where is the problem.

**Pull request merged**